components/PostCardList/PostCardList.stories.tsx

```diff
@@ -0,0 +1,120 @@
+import { action } from "@storybook/addon-actions";
+import { select } from "@storybook/addon-knobs";
+import * as React from "react";
+import GameSituation, { GameType } from "../../models/GameSituation";
+import { Suit, Rank } from "../../models/PlayingCard";
+import Post, { PostId, PostTitle, PostBody } from "../../models/Post";
+import { UserId } from "../../models/User";
+import PostCardList from "./PostCardList";
+import PostCardListItem from "./PostCardListItem";
+import { PostType } from "./PostTypeContext";
+
+export default {
+  title: "PostCardList",
+  component: PostCardList,
+  subcomponent: { PostCardListItem }
+};
+
+export const example = () => {
+  return (
+    <PostCardList postType={select("postType", POST_TYPES, PostType.recent)}>
+      <PostCardListItem post={post} onClick={action("onClick")} />
+      <PostCardListItem post={post} onClick={action("onClick")} />
+      <PostCardListItem post={post} onClick={action("onClick")} />
+      <PostCardListItem post={post} onClick={action("onClick")} />
+      <PostCardListItem post={post} onClick={action("onClick")} />
+      <PostCardListItem post={post} onClick={action("onClick")} />
+    </PostCardList>
+  );
+};
+
+export const postCardListItem = () => (
+  <PostCardList postType={select("postType", POST_TYPES, PostType.recent)}>
+    <PostCardListItem post={post} onClick={action("onClick")} />
+  </PostCardList>
+);
+
+export const postType = () => (
+  <>
+    <PostCardList postType={PostType.recent}>
+      <PostCardListItem post={post} onClick={action("onClick")} />
+    </PostCardList>
+    <PostCardList postType={PostType.popular}>
+      <PostCardListItem post={post} onClick={action("onClick")} />
+    </PostCardList>
+  </>
+);
```

Comment:
I think we cannot understand what this example means on Storybook.

<img width="1060" alt="Screen Shot 2020-03-29 at 3 15 12 PM" src="https://user-images.githubusercontent.com/4289883/77862483-751a1000-71d0-11ea-8bfa-a0daf22b9f03.png">

It would be better if its name is `showLastUpdate` instead of `postType`. Because in the default behavior (it's on `postType`'s default value), list items show the posted date.